Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM circleci/node:10.14.1 as builder
+
+WORKDIR /usr/src/app
+
+COPY /src /usr/src/app/src
+COPY /public /usr/src/app/public
```

Comment:
this shouldn't be needed as the fe is hosted separately

Reply:
Not sure I understood, but I need to clarify that this is the frontend.
The backend is hosted at https://github.com/teamdigitale/io-onboarding-pa-api